Annotato/Annotato/Views/Document/Edit/Annotation/AnnotationView.swift

```diff
@@ -125,6 +129,26 @@ class AnnotationView: UIView {
                 }
             }
         }.store(in: &cancellables)
+
+        viewModel.$isResolving.sink { [weak self] isResolving in
+            guard !isResolving else {
+                return
+            }
+            guard let self = self else {
+                return
+            }
+            if self.viewModel.resolveBySave {
+                let mergeConflictsHeight = self.mergeConflictsPalette.height
+                self.mergeConflictsPalette.resetDimensions()
+                self.mergeConflictsPalette.removeFromSuperview()
```

Comment:
Why is this handled from the view rather than the view model? Couldn't AnnotationViewModel directly tell mergeConflictPaletteViewModel to disappear?

Reply:
I don't understand this part
I need to call removeFromSuperview() which is traditionally done in the view layer is it? Because I need a reference to the view itself